AudioKitSynthOne/Keyboard/KeyboardSettingsViewController.swift

```diff
@@ -75,3 +75,22 @@ class KeyboardSettingsViewController: UIViewController {
 
     }
 }
+
+/**
+Accessibility Functionality
+*/
+extension KeyboardSettingsViewController {
+	func setUpAccessibility() {
+
+		keyboardModeSegment.subviews[0].accessibilityLabel = NSLocalizedString("White", comment: "White")
+		keyboardModeSegment.subviews[2].accessibilityLabel = NSLocalizedString("Dark", comment: "Dark")
+		
+		octaveRangeSegment.subviews[0].accessibilityLabel = NSLocalizedString("1", comment: "1")
+		octaveRangeSegment.subviews[1].accessibilityLabel = NSLocalizedString("2", comment: "2")
+		octaveRangeSegment.subviews[2].accessibilityLabel = NSLocalizedString("3", comment: "3")
+		
+		labelModeSegment.subviews[0].accessibilityLabel = NSLocalizedString("None", comment: "None")
+		labelModeSegment.subviews[1].accessibilityLabel = NSLocalizedString("C", comment: "C")
+		labelModeSegment.subviews[2].accessibilityLabel = NSLocalizedString("All", comment: "All")
+	}
```

Comment:
Could use a bit more context in the comments here, for the translators' sake, but I'm nit-picking :)
Also not entirely sure it was necessary to localize just numbers, but doesn't really matter I guess...

Reply:
That's to turn on/off the label notes on the keyboard. Is that useful for accessibility? Adding additional visual labeling?

Reply:
Mostly these values are used to be read by the VoiceOver when hitting the buttons. The whole interface must be readable.

Reply:
Megastep, is correct.  There are exceptions, but in general you try and get everything pertinent for VoiceOver.   It is not just those which have complete vision loss which use VO, but people with low vision or physical/cognitive impairments need it as well. Sort of as a way to double check the control they are trying to use is the correct one.

Then you have the weirdos like me, who can see fine but use it so I don't have to look at the phone.